scanner/src/lib/dataproducts/Battery_CarbonFootprint_v0.1.svelte

```diff
@@ -0,0 +1,288 @@
+  .chart-wrapper {
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    margin-bottom: 1rem;
+
+    p {
+      font-size: 0.75rem;
+    }
+
+    :global(svg) {
+      max-width: 25rem;
+      margin: 0 auto;
+    }
+  }
+
+  $animation-speed: 650ms;
+
+  :global(.chart-wrapper.animate svg #slice-0) {
+    animation: scale0 $animation-speed ease-in-out;
+
+    @keyframes scale0 {
+      50%,
+      100% {
+        transform: scale(1);
+        transform-origin: center;
+      }
+      75% {
+        transform: scale(1.02);
+        transform-origin: center;
+      }
+    }
+  }
+
+  :global(.chart-wrapper.animate svg #slice-1) {
+    animation: scale1 $animation-speed ease-in-out;
+
+    @keyframes scale1 {
+      0%,
+      50%,
+      100% {
+        transform: scale(1);
+        transform-origin: center;
+      }
+      25% {
+        transform: scale(1.04);
+        transform-origin: center;
+      }
+    }
+  }
+
+  :global(.chart-wrapper.animate svg #slice-2) {
+    animation: scale2 $animation-speed ease-in-out;
+
+    @keyframes scale2 {
+      10%,
+      60%,
+      100% {
+        transform: scale(1);
+        transform-origin: center;
+      }
+      35% {
+        transform: scale(1.04);
+        transform-origin: center;
+      }
+    }
+  }
+
+  :global(.chart-wrapper.animate svg #slice-3) {
+    animation: scale3 $animation-speed ease-in-out;
+
+    @keyframes scale3 {
+      20%,
+      70%,
+      100% {
+        transform: scale(1);
+        transform-origin: center;
+      }
+      45% {
+        transform: scale(1.04);
+        transform-origin: center;
+      }
+    }
+  }
+
+  :global(.chart-wrapper.animate svg #slice-4) {
+    animation: scale4 $animation-speed ease-in-out;
+
+    @keyframes scale4 {
+      30%,
+      80%,
+      100% {
+        transform: scale(1);
+        transform-origin: center;
+      }
+      55% {
+        transform: scale(1.04);
+        transform-origin: center;
+      }
+    }
+  }
+
+  :global(.chart-wrapper.animate-second svg #slice-0) {
+    animation: scale00 $animation-speed ease-in-out;
+
+    @keyframes scale00 {
+      0%,
+      50%,
+      100% {
+        transform: scale(1);
+        transform-origin: center;
+      }
+      25% {
+        transform: scale(1.02);
+        transform-origin: center;
+      }
+    }
+  }
+
+  :global(.chart-wrapper.animate-second svg #slice-1) {
+    animation: scale01 $animation-speed ease-in-out;
+
+    @keyframes scale01 {
+      10%,
+      60%,
+      100% {
+        transform: scale(1);
+        transform-origin: center;
+      }
+      35% {
+        transform: scale(1.04);
+        transform-origin: center;
+      }
+    }
+  }
+
+  :global(.chart-wrapper.animate-second svg #slice-2) {
+    animation: scale02 $animation-speed ease-in-out;
+
+    @keyframes scale02 {
+      20%,
+      70%,
+      100% {
+        transform: scale(1);
+        transform-origin: center;
+      }
+      45% {
+        transform: scale(1.02);
```

Comment:
These small slices were supposed to use 1.04

```suggestion
        transform: scale(1.04);
```